.github/workflows/cirrus_status.yml

```diff
@@ -42,35 +40,38 @@ jobs:
                   status=$(jq --raw-output \
                         '.[] | select(.name == "Total Success") | .status' \
                         "$ccirjson")
+                  branch=$(jq --raw-output \
+                        '.[] | select(.name == "Total Success") | .branch' \
+                        "$ccirjson")
 
                   if [[ -n "$prn" ]] && \
                      [[ "$prn" != "null" ]] && \
-                     [[ $prn -gt 0 ]] && \
-                     [[ "$status" == "COMPLETED" ]]
+                     [[ $prn -gt 0 ]]
                   then
                       printf "prn=%s\n" "$prn" >> $GITHUB_OUTPUT
-                      printf "bid=%s\n" "$bid" >> $GITHUB_OUTPUT
                       printf "is_pr=%s\n" "true" >> $GITHUB_OUTPUT
                   else
                       printf "prn=%s\n" "0" >> $GITHUB_OUTPUT
-                      printf "bid=%s\n" "0" >> $GITHUB_OUTPUT
                       printf "is_pr=%s\n" "false" >> $GITHUB_OUTPUT
                   fi
+                  printf "bid=%s\n" "$bid" >> $GITHUB_OUTPUT
+                  printf "status=%s\n" "$status" >> $GITHUB_OUTPUT
+                  printf "branch=%s\n" "$branch" >> $GITHUB_OUTPUT
 
-            - if: steps.retro.outputs.is_pr == 'true'
+            - if: &is_successful_pr steps.retro.outputs.is_pr == 'true' && steps.retro.outputs.status == 'COMPLETED'
```

Comment:
Cool syntax, didn't know this was possible!

Reply:
that is normal yaml syntax https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases that we use for cirrus yaml all the time, I have however not tested running that with github.

And now that you didn't know or come across that makes me second guess the github support for it and a quick search later I land at https://github.com/actions/runner/issues/1182 which seems to suggest the github parser does NOT support it :(

So I guess I duplicate that string then, not the a big deal